sematic/examples/lightning_resnet/main.py

```diff
@@ -0,0 +1,111 @@
+# Standard Library
+import argparse
+import logging
+from dataclasses import replace
+
+# Sematic
+import sematic
+from sematic.ee.ray import RayNodeConfig
+from sematic.examples.lightning_resnet.pipeline import pipeline
+from sematic.examples.lightning_resnet.train_eval import (
+    DataConfig,
+    EvaluationConfig,
+    TrainingConfig,
+    TrainLoopConfig,
+)
+from sematic.types.types.aws.s3 import S3Location
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+CHECKPOINT_LOCATION = S3Location.from_uri("s3://sematic-dev/lightning-resnet")
```

Comment:
Please make this configurable through an arg parameter, so that users outside the company can run this.

Reply:
done